src/lib/cimbar_js/cimbar_js.cpp

```diff
@@ -136,7 +154,9 @@ int configure(unsigned color_bits, unsigned ecc, int compression, bool legacy_mo
 	// make sure we've initialized
 	int window_size_x = cimbar::Config::image_size_x() + 16;
 	int window_size_y = cimbar::Config::image_size_y() + 16;
-	initialize_GL(window_size_x, window_size_y);
+	int initRes = cimbare_init_window(window_size_x, window_size_y);
+	if (initRes < 0)
+		return initRes;
```

Comment:
Good to know if it blows up in our face. This function is how the JS interacts with init_window.